channel-transport-reaction/chemical-fenics/requirements.txt

```
@@ -1,3 +1,5 @@
 numpy >1, <2
 pyprecice~=3.0
-scipy
+mpi4py < 4
+fenicsprecice
```

```suggestion
fenicsprecice~=2.0
```
Be aware that for `fenicsprecice` we had a breaking release consistently with preCICE 3.0.0. Therefore, we should not accept `fenicsprecice<2.0`.